docker/templates/new-gatsby-app/{{cookiecutter.app_name}}/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
Nice! We should deprecate [this guidance](https://github.com/datamade/testing-guidelines/blob/master/intro-to-javascript-testing.md#jshint) after we bring this in.

Reply:
Good point!

Reply:
It'd be nice to bring this `testing-guidelines` repo into `how-to`... what do you think about the sequencing of tasks here?

Reply:
We decided to leave the external repos (data-analysis-guidelines, data-making-guidelines, testing-guidelines, deploy-a-site before it was deprecated) separate, to preserve any external links to them. (data-making-guidelines, in particular, has a life of its own.) I think that, after this comes in, we should deprecate the JavaScript testing portion of testing guidelines, since it was never really adopted anyway, but leave everything else over there for now.

Reply:
Sounds good. I can set up a PR with that deletion.